src/sqlodbc/opensearch_info.cpp

```diff
@@ -652,12 +652,13 @@ void GenerateColumnQuery(std::string &query, const std::string &table_name,
                          const std::string &column_name, const bool table_valid,
                          const bool column_valid, const UWORD flag) {
     bool search_pattern = (~flag & PODBC_NOT_SEARCH_PATTERN);
-    query = "DESCRIBE TABLES LIKE ";
+    query = "DESCRIBE TABLES LIKE '";
     query += table_valid
                  ? (search_pattern ? table_name : "^" + table_name + "$")
                  : "%";
+    query += '\'';
```

Comment:
It'd be nicer if you didn't mix n match different quote types in the function.

Reply:
In terms of C++ performance, `str + char` is faster than `str + str` and consumes less memory.